redash/query_runner/axibase_tsd.py

```diff
@@ -0,0 +1,191 @@
+from io import StringIO
+import json
+import logging
+import sys
+import uuid
+
+import pandas as pd
+import atsd_client
+from atsd_client.exceptions import SQLException
+from atsd_client.services import SQLService, MetricsService
```

Comment:
All the ATSD imports should be in a try/except block along with an enabled flag. 

See [here](https://github.com/getredash/redash/blob/master/redash/query_runner/mssql.py#L11-L15) (and [here](https://github.com/getredash/redash/blob/master/redash/query_runner/mssql.py#L75-L77)) for example.